portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Jen!</h1>");
```

Comment:
Make sure this change is not part of this PR. Make is part of the next PR. To do this you would have to branch out.

Reply:
Or since this PR is about to merged soon, just remove this change, make other fixes, merge, then add this line.